libsawtooth/Cargo.toml

```diff
@@ -25,13 +25,16 @@ description = """\
 
 [dependencies]
 cbor-codec = { version = "0.7", optional = true }
+futures = { version = "0.1", optional = true }
 hex = "0.4"
 log = "0.4"
 lmdb-zero = { version = "0.4", optional = true }
 metrics = { version = "0.12", features = ["std"], optional = true }
 openssl = { version = "0.10", optional = true }
 protobuf = "2"
 redis = { version = "0.13.0", default-features = false, optional = true }
+reqwest = { version = "0.10", optional = true, features = ["blocking", "json"] }
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
Should this be optional and pulled in with 'client'?

Reply:
Fixed